### PR TITLE
Add Swimat.app 1.0

### PR DESCRIPTION
**Pull request by Jintin**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/swimat.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'swimat' do
+  version '1.0'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/swimat.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/swimat.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+  url "https://github.com/Jintin/Swimat/releases/download/v#{version}/Swimat.zip"
```
Comment (vitorgalvao) [inline Casks/swimat.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/swimat.rb (hunk 1 above)
Comment (Jintin) [inline Casks/swimat.rb]: I've update it. Thanks for remind.

**Comment by Jintin** [comment]
I've add `sha256` and `appcast`. 
Thanks for remind.

**Pull request merged**